stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1150,23 +1150,18 @@ public func _TFCCreateCTensorHandle<T>(_ value : T,
 // - MARK: Dynamic compilation (per-op dispatch) entrypoints
 //===----------------------------------------------------------------------===//
 
-// TODO: Remove the RunXXXTest method with hard-coded tensor computation.
 @inlinable
-@_silgen_name("_swift_tfc_RunEagerConstTest")
-public func _RunEagerConstTest(_ ctx: CTFEContext) -> TensorHandle<Float> {
-  debugLog("Calling _RunEagerConstTest()")
-  let cHandle: CTensorHandle! = TFE_RunConstOp(ctx)
-  return TensorHandle<Float>(owning: cHandle)
-}
-
-@inlinable
-@_silgen_name("_swift_tfc_GetGlobalEagerContext")
+@_cdecl("_swift_tfc_GetGlobalEagerContext")
 public func _GetGlobalEagerContext() -> CTFEContext {
```

Comment:
I don’t think these have to be public. @usableFromInline should work fine. Have you tried it?

Reply:
Yes this works for the `@_cdecl` functions. Changed.